docs/contrib/GettingInvolved.md

```diff
@@ -0,0 +1,84 @@
+[OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/).
+In the
+[flow directory](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow)
+you will see a directory for
+[designs](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/designs)
+with Makefiles to run them, and one for PDK
+[platforms](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/platforms)
+used by the designs. If you add a new PDK platform be sure to add at
+least one design that uses it.
+
+## Contributing Scripts and Code
+
+We follow the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html).
+If you find code that is not following this guide, within each file that
+you edit, follow the style in that file. Please pay careful attention to the
+[tool checklist](DeveloperGuide.md#Tool Checklist) for all code. If you want
+to add or improve functionality in OpenROAD please start with the top
+level [app](https://github.com/The-OpenROAD-Project/OpenROAD/) repo. You
+can see in the src directory that submodules exist pointing to tested
+versions of the other relevant repos in the project. Please look at the
+tool workflow in the developer guide [document](DeveloperGuide.md)
+to work with the app and its submodule repos in an efficient way.
+
+Please pay attention to the
+[test directory](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/test)
+and be sure to add tests for any code changes that you make with open
+sourceable PDK and design information. We provide the nandgate45 PDK in
+the OpenROAD-flow-scripts repo to help with this. Pull requests with
+code changes are unlikely to be accepted without accompanying test
+cases. There are many
+[examples](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/test/gcd_nangate45.tcl)
+tests. Each repo has a test directory as well with tests you should run
+and add to if you modify something in one of the submodules.
+
+For changes that claim to improve QoR or PPA, please run many tests and
+ensure that the improvement is not design specific. There are designs in
+the
+[OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/)
+repo which can be used unless the improvement is technology specific.
+
+Do not add runtime or build dependencies without serious thought. For a
+project like OpenROAD with many application sub components, the software
+architecture can quickly get out of control. Changes with lots of new
+dependencies which are not necessary are less likely to be integrated.
+
+If you want to add TCL code to define a new tool command look at pdngen
+as an example of how to do so. Take a look at the
+[cmake file](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/CMakeLists.txt)
+which automatically sources the tcl code and the
+[tcl code](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/pdn/src/PdnGen.tcl)
+itself.
+
```

Comment:
Include DCO certification requirements for contributors in a separate heading.

Reply:
Fixed on 88fd131.